openff/toolkit/tests/test_utils.py

```diff
@@ -133,3 +134,13 @@ def test_sort_smirnoff_dict():
     assert smirnoff_dict == OrderedDict(
         sort_smirnoff_dict(forcefield._to_smirnoff_data())
     )
+
+
+def test_import_message_exception_raises_warning(caplog):
+    # TODO: Remove when removing MessageException
+    deprecation_msg = (
+        "DEPRECATED and will be removed in a future release of the OpenFF Toolkit"
+    )
+
+    with pytest.warns(DeprecationWarning, match=deprecation_msg):
+        from openff.toolkit.utils.exceptions import MessageException
```

Comment:
You could also add a test that importing something else does *not* raise a warning.

```
with pytest.warns(None) as rec:
	from openff.toolkit.utils.exceptions import SMILESParseError
assert len(rec) == 0  # no warnings
```